src/azure-cli-core/azure/cli/core/utils/vcr_test_base.py

```diff
@@ -23,11 +23,14 @@
 import jmespath
 from six import StringIO
 
-import azure.cli as cli
+# TODO Should not depend on azure.cli.main package here.
+# Will be ok if this test file is not part of azure.cli.core.utils
```

Comment:
Perhaps test should not be part of CLI core? Possibly part of extensions (assuming extensions are removed from the core)